src/sidebar/store/modules/selection.js

```diff
@@ -1,42 +1,34 @@
-/**
- * This module handles the state affecting the visibility and presence of
- * annotations and threads in the UI.
- */
```

Comment:
Comments about what a store module is for are better attached to the ES module's main export. That said, I don't think this comment really adds value beyond looking at the `State` type in the module, so I didn't re-add it.

Reply:
Hmmm, I do remember in the past being confused about what "selection" means. But I suppose `state` is a lot easier to scan at a glance these days...